scripts/results/arrange-data.py

```diff
@@ -162,41 +187,54 @@ def find_first_match(user_input):
 session_num = 0
 options = []
 Months = {
+    'jan': 1,
     'Jan': 1,
+    'feb': 2,
     'Feb': 2,
+    'mar': 3,
     'Mar': 3,
+    'apr': 4,
     'Apr': 4,
+    'may': 5,
     'May': 5,
+    'jun': 6,
     'Jun': 6,
+    'jul': 7,
     'Jul': 7,
+    'aug': 8,
     'Aug': 8,
+    'sep': 9,
     'Sep': 9,
+    'oct': 10,
     'Oct': 10,
+    'nov': 11,
     'Nov': 11,
+    'dec': 12,

```

Comment:
Not a good implementation. just keep the lowercase months and for the user input, make sure that you convert it to lowercase (e.g., `s.lower()`) before matching it to the months.
This way all situations will be covered. For example, right now if the user input is `dEc` it won't match any of the months.